features/mesh/gateways/Index.feature

```diff
@@ -59,6 +59,7 @@ Feature: mesh / gateways / index
   Rule: The Gateway listing can filter gateways by type
 
     Scenario: No filtering
+      Then the "$item" element exists 2 times
```

Comment:
This ensures that that we wait until enough rendering has finished (but no longer) to assert whether the below request was called.

Reply:
Once I'd reimplemented the request asserting this was one I could remove.